frontend/src/pages/FormStaffPage.vue

```diff
@@ -77,10 +81,10 @@ const clientTypesList: CodeNameType[] = [
     code: "F",
     name: "Ministry of Forests",
   },
-  {
+  ...(isAdminInd ? [{
```

Comment:
A better approach would be to push the entry later instead of filtering inline here. It would make the code more readable